package.json

```diff
@@ -50,8 +50,8 @@
     "prestart": "yarn downloadCustomTypes",
     "start": "PORT=3001 react-app-rewired start",
     "build": "react-app-rewired build",
-    "test:e2e": "jest -c e2e/jest.config.js",
-    "test:e2e-w": "jest -c e2e/jest.config.js --watch",
+    "test:e2e": "jest -c e2e/jest.config.js --detectOpenHandles",
```

Comment:
I think `--detectOpenHandles` should be used only when we want to identify leaks (and Jest informs about them anyway) not in each run

Reply:
What's the drawback of using it always?

Reply:
2 drawbacks:
a) It implies `--runInBand`, making tests run serially.
b) It has a significant performance penalty

Documentation proposes that this should only be used for debugging.

Reply:
Right. It is not needed .. i have removed the parameter.